pkg/gateway/api/api.go

```diff
@@ -962,7 +922,6 @@ func (s *gatewayService) registerEvents(w http.ResponseWriter, req *http.Request
 					Message:   err.Error(),
 				}
```

Comment:
Though this is not a change in this PR, I think `continue` is needed in the line.